runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -493,6 +481,13 @@ public void onMenuOptionClicked(MenuOptionClicked event)
 		}
 
 		TeleportWidget teleportWidget = TeleportWidget.of(event.getParam1());
+		// check for Group Iron Man Helm Teleport
+		if (event.getMenuOption().contains("Teleport")
+				&& event.getMenuTarget().toLowerCase(Locale.ROOT).contains("group iron helm"))
```

Comment:
This should be checking the menu type and the clicked item ID instead of the target text.

Reply:
Do you man with `event.getId()`?

Reply:
log.info("EventId: " + event.getId());
        log.info("Param0: " + event.getParam0());
        log.info("Param1 : " + event.getParam1());
        log.info("ActionParam: " + event.getActionParam());
Hmm... none of these seem to return the item id when the item is equipped 

Maybe I am missing something but it doesn't look like any of the other fields would return the item id